README.md

```diff
@@ -1,19 +1,22 @@
 # DataComp-LM (DCLM)
 
 ## Table of Contents
-- [Introduction](#introduction)
-- [Leaderboard](#leaderboard)
-- [Getting Started](#getting-started)
-- [Selecting Raw Sources](#selecting-raw-sources)
-- [Processing the Data](#processing-the-data)
-- [Deduplication](#deduplication)
-- [Tokenize and Shuffle](#tokenize-and-shuffle)
-- [Model Training](#model-training)
-- [Evaluation](#evaluation)
-- [Submission](#submission)
-- [Contributing](#contributing)
-- [How to Cite Us](#how-to-cite-us)
-- [License](#license)
+- [DataComp-LM (DCLM)](#datacomp-lm-dclm)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I don't think the ToC is needed in the ToC.. 
also, isn't there a redundant tab for all items here?